code/crates/core-consensus/src/handle/driver.rs

```diff
@@ -308,10 +309,7 @@ where
                 "Decided",
             );
 
-            perform!(
-                co,
-                Effect::ScheduleTimeout(Timeout::commit(consensus_round), Default::default())
-            );
+            try_decide(co, state, metrics).await?;
```

Comment:
This likely require more in-depth review that it's the right way to do things.

Reply:
This seems correct to me. The name `try_decide` looks strange. 
We also need to remove the other calls from the handlers in `sync.rs` and `proposed_value.rs`, the only reason we had them there was to skip the commit timer in case we were syncing.